shiny/ui/dataframe/js/build.ts

```diff
@@ -0,0 +1,27 @@
+import { build } from "esbuild";
+import { sassPlugin } from "esbuild-sass-plugin";
+import * as fs from "node:fs";
+
+async function bundle() {
+  try {
+    const options = {
+      entryPoints: ["index.tsx"],
+      bundle: true,
+      outdir: "dist",
+      minify: true,
+      sourcemap: true,
+      plugins: [sassPlugin({ type: "css-text" })],
+      target: ["safari12"],
```

Comment:
Any particular reason you chose `safari12` here?

Reply:
Ummmm. I don't remember. I think I looked at what versions of iOS were supported by what iPhones, and therefore what version of Safari a not-too-old phone might be stuck on? Sorry, I should have documented that decision. What have you been setting it to?

Reply:
In stuff I've been working on, I haven't specified the `target`, and it apparently defaults to `"esnext"`. I don't know if that's a problem in practice or not.

https://esbuild.github.io/api/#target